generate.js

```diff
@@ -341,40 +350,62 @@ function downloadFromUrl(srcUrl, appendUrl, apiCache, cacheKey, onEachComplete,
         });
     });
 }
-/////////////////////////////////// Major step 3 - Generate the indicated ouptut files ///////////////////////////////////
-function generateApis(buildIdentifier, targetOutputPathList, buildFlags, apiSrcDescription) {
-    console.log("Generating PlayFab APIs from specs: " + apiSrcDescription);
+function generateApis(buildIdentifier, target) {
+    console.log("Generating PlayFab APIs from specs: " + sdkGeneratorGlobals.apiSrcDescription);
+    var genConfig = null;
+    var genConfigPath = path.resolve(target.destPath, "genConfig.json");
+    try {
+        genConfig = require(genConfigPath);
+        console.log("Loaded genConfig at: " + genConfigPath);
+    }
+    catch (_) {
+        console.log("Did not find: " + genConfigPath);
```

Comment:
>console.log("Did not find: " + genConfigPath); [](start = 8, length = 46)

Should we allow the generation to continue to the code below if something went wrong here?

Reply:
This file is explicitly designed to be optional (for now). Basically, we want a certain set of inputs, either from command line, or environment variables, or from genConfig.json.
If we get the right inputs, we don't care where they came from.